src/main/resources/application.properties

```diff
@@ -1,4 +1,6 @@
 spring.application.name=Commission Calculator
 spring.datasource.url=jdbc:postgresql://localhost:5432/sales_commission
 spring.datasource.username = postgres
-spring.datasource.password = password
\ No newline at end of file
+spring.datasource.password = password
+
+spring.jpa.generate-ddl=true
```

Comment:
Based on what does this generate the ddl?

Reply:
The table names with fields that have snake_case, in this case in the table Sales, person_id and date_stamp, were not being read, and I could not do any CRUP operations.

Reply:
Hmm, that shouldn't require jpa settings... it sounds like something else is wrong

Reply:
Yes, i did not need the settings. Not sure what was wrong, probably it was something as silly as typo. 
To double check, and for consistency I refactored `surname` to `last_name` and still works fine.